src/export/InstanceExporter.ts

```diff
@@ -51,14 +51,27 @@ export class InstanceExporter {
             .join('.');
           rulePathPart += '.';
 
-          this.setFixedValuesForDirectChildren(
+          this.setFixedValuesFromStructureDefinition(
             element,
             rulePathPart,
             instanceDef,
             instanceOfStructureDefinition
           );
         }
+      } catch (e) {
+        logger.error(e.message, rule.sourceInfo);
+      }
+    });
 
+    // All rules will be FixValueRule
```

Comment:
I feel like the comments on line 30 and this line (60) maybe could be more specific about what's happening.  I.e.,

```
30: // Fix all values from the SD that are exposed when processing the instance rules

66: // Fix all values explicitly set in the instance rules (all rules will be FixValueRule)
```
Or something like that...